src/applications/personalization/profile/components/Profile.jsx

```diff
@@ -134,11 +134,8 @@ class Profile extends Component {
           dismissDowntimeWarning={this.props.dismissDowntimeWarning}
           initializeDowntimeWarnings={this.props.initializeDowntimeWarnings}
           messaging={{
-            title: (
-              <h3>
-                Some parts of the profile will be down for maintenance soon
-              </h3>
-            ),
+            title:
+              'Some parts of the profile will be down for maintenance soon',
```

Comment:
In doing this work, I was trying to simulate downtime approaching and downtime being active, and discovered that the title needs to be a string instead of a full dom node, and passing a full dom node was actually causing this modal to not load correctly.